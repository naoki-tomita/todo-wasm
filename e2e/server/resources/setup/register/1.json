{ "text": "Todoを登録できるようにする" }
