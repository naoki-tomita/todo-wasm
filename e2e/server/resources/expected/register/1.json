{ "text": "Todoを登録できるようにする", "done": true }
